schema/src/main/jade-tables/hles_owner.table.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "hles_owner",
+  "columns": [
+    {
+      "name": "owner_id",
+      "datatype": "integer",
+      "type": "primary_key"
+    },
+    {
+      "name": "od_age_range_years",
+      "datatype": "string"
+    },
+    {
+      "name": "od_max_education",
+      "datatype": "string"
+    },
+    {
+      "name": "od_max_education_other",
+      "datatype": "string"
+    },
+    {
+      "name": "od_race",
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "od_race_other",
+      "datatype": "string"
+    },
+    {
+      "name": "od_hispanic",
+      "datatype": "boolean"
+    },
+    {
+      "name": "od_annual_income_range_usd",
+      "datatype": "string"
+    },
+    {
+      "name": "oc_household_adult_count",
+      "datatype": "string"
+    },
```

Comment:
@danxmoran in the mapping doc, there is also an `oc_household_person_count` column. You left that out intentionally, right?

Reply:
Nope, good catch

Reply:
Can you add it and push to the branch? Or fix it when working on the mapping implementation? Deep in the WIP weeds on the dog transformations